Enterprise-CMCS/eAPD
# 4556 Updates to export/read-only proposed budget section

Resolves #4556 

### Description
Updates the export/read-only view to not show the help text and alert for the proposed budget section

### Chromatic Link
https://www.chromatic.com/pullrequest?appId=61d5b948cf6f17003a12bf77&number=4674

### Significant changes or possible side effects
None

### Automated test cases written
| Given | When | Then | Type (jest, tap, cypress) |
| ----- | ---- | ---- | ------------------------- |
| an MMIS apd | viewing the read-only export page proposed budget section | should render properly | enzyme snapshot |

### Steps to manually verify this change

1. Login and view a MMIS apd
2. Navigate to the read-only/export view
3. Verify the help text and alert box is no longer displayed in the proposed budget section

### This pull request is ready to code review when

      tests, refactor
      screenreader, text scaling) OR an exemption is documented

### This pull request is ready to test when


### This pull request is ready to review when QA has


### This pull request can be merged when
